src/address.ts

```diff
@@ -0,0 +1,9 @@
+export interface AddressInput {
+  CEP: string;
```

Comment:
```suggestion
  cep: string;
```

No código, costuma ser melhor manter a convenção de case do que a do idioma (camelCase > usar tudo maiúsculo em sigla, por exemplo).